launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -1014,8 +1014,18 @@ QMap<QString, QString> MinecraftInstance::createCensorFilterFromSession(AuthSess
 
 MessageLevel::Enum MinecraftInstance::guessLevel(const QString& line, MessageLevel::Enum level)
 {
-    QRegularExpression re("\\[(?<timestamp>[0-9:]+)\\] \\[[^/]+/(?<level>[^\\]]+)\\]");
-    auto match = re.match(line);
+    if (line.contains("overwriting existing"))
+        return MessageLevel::Fatal;
+
+    // NOTE: this diverges from the real regexp. no unicode, the first section is + instead of *
```

Comment:
Shouldn't this check be last?
It should apply if another level wasn't directly specified by the line IMO
Some errors are emitted ass warnings only.